app/javascript/packs/banner_editor.jsx

```diff
@@ -421,6 +423,46 @@ export default class BannerEditor extends React.Component {
       }
   }
 
+  tipToOption(){
```

Comment:
Think we could remove this option if we set a `donationAmountIndex` in the state? I think that might make more sense